test/manual/atlas_connectivity.test.js

```diff
@@ -15,8 +15,15 @@ describe('Atlas Connectivity', function() {
     context(configName, function() {
       CONFIGS[configName].forEach(connectionString => {
         const name = connectionString.indexOf('mongodb+srv') >= 0 ? 'mongodb+srv' : 'normal';
-        it(`${name} (unified)`, makeConnectionTest(connectionString, { useUnifiedTopology: true }));
-        it(`${name} (legacy)`, makeConnectionTest(connectionString, { useUnifiedTopology: false }));
+        // TODO: Skipping until NODE-3308 is merged.
```

Comment:
Good call skipping these too, but NODE-3308 is actually referring to different tests: the `A successful find event with a getmore and the server kills the cursor` failures.

Reply:
All tests passing now.